src/language-js/comments.js

```diff
@@ -831,6 +845,31 @@ function handleVariableDeclaratorComments(
   return false;
 }
 
+function handleTSFunctionTrailingComments(
+  text,
+  enclosingNode,
+  followingNode,
+  comment,
+  options
+) {
+  if (
+    followingNode ||
+    !enclosingNode ||
+    (enclosingNode.type !== "TSMethodSignature" &&
+      enclosingNode.type !== "TSDeclareFunction" &&
+      enclosingNode.type !== "TSAbstractMethodDefinition") ||
+    privateUtil.getNextNonSpaceNonCommentCharacter(
+      text,
+      comment,
+      options.locEnd
+    ) !== ";"
+  ) {
```

Comment:
Please invert this condition, so that this function was more similar to other `handle...Comments` functions in this file, with `return false` in the end.